src/main/java/mods/railcraft/common/blocks/multi/TileMultiBlockOven.java

```diff
@@ -174,8 +170,8 @@ public int getLightValue() {
     }
 
     private void processActions() {
-        paused = actions.stream().anyMatch(a -> a == Actions.PAUSE);
-        actions.clear();
+//        paused = actions.stream().anyMatch(a -> a == Actions.PAUSE);
+//        actions.clear();
```

Comment:
Why is the action code commented out?

Reply:
Because I think they may be useless in the immediate future

Reply:
K

Reply:
Will restore it.